### DISCUSSION TITLE
Can you add IDX for relay for domoticz more than 4?

### DISCUSSION THREAD

**Discussion by deasonsv** [arendst/Tasmota#20154] [Q&A] [answered]
Hi. Can you add IDX for relay for domoticz more than 4? For my project need 6 separate IDX

**Comment by arendst** [comment, marked as answer]
Learn to use rules (https://tasmota.github.io/docs/Rules/)

Example: ``rule1 on power5#state do dzsend1 9001,%value% endon on power6#state do dzsend1 9002,%value% endon``

where 9001 and 9002 are the domoticz indexes.

And do not forget to enable the rule with command ``rule1 1``
